wagtail/admin/rich_text/editors/draftail/__init__.py

```diff
@@ -63,9 +64,22 @@ def format_value(self, value):
 
         return self.converter.from_database_format(value)
 
+    def get_options(self):
+        chooserUrls = {
+            "pageChooser": reverse("wagtailadmin_choose_page"),
+            "externalLinkChooser": reverse("wagtailadmin_choose_page_external_link"),
+            "emailLinkChooser": reverse("wagtailadmin_choose_page_email_link"),
+            "phoneLinkChooser": reverse("wagtailadmin_choose_page_phone_link"),
+            "anchorLinkChooser": reverse("wagtailadmin_choose_page_anchor_link"),
+            "embedsChooser": reverse("wagtailembeds:chooser"),
+            "documentChooser": reverse("wagtaildocs_chooser:choose"),
+            "imageChooser": reverse("wagtailimages_chooser:choose"),
```

Comment:
As noted in the other feedback, we cannot assume that non-core modules are installed (documents, images, embeds) so we will need to find another way to get these urls IF they are enabled.

This is where it's a bit tricky but possibly finding all classes that are chooser viewsets OR setting up a registry on the chooser viewset class of URLs and then reading that registry here.

The page, external link, email, phone, anchor are OK to be hard-coded here though as these are code ones and exist hard-coded in the templates today.

Reply:
Thanks ,I have made the changes but I just surprised because you ask me to remove ` editor_js` hooks 

> A reminder that you will want to remove the editor_js hooks used by embeds/documents/images addition of the 
   globals in the

and also here you ask me to check about global I just want to know where `global` come from ?
I already removed the  editor_js` hooks as you ask me to do. or global come from third-party packages ? ,
really I don ,t know, but I made the changes .

Reply:
yes , but I just want  third-party packages to reproduce this issue on it , but 
package compatible with the newer version of wagtail , because I get error when I tried to install 
https://github.com/themotleyfool/wagtail-draftail-snippet , and I had to uninstall and reinstall wagtail again .

Reply:
I have added a comment that will help a bit more with this, I think we can take a different approach.

https://github.com/wagtail/wagtail/issues/11586#issuecomment-1938165894